cmake/guidelineSupportLibrary.cmake

```diff
@@ -4,14 +4,21 @@
 # for multiple versions of cmake.
 #
 # Any functions/macros should have a gsl_* prefix to avoid problems
-if (DEFINED guideline_support_library_include_guard)
-    return()
+if (CMAKE_VERSION VERSION_GREATER 3.10 OR CMAKE_VERSION VERSION_EQUAL 3.10)
```

Comment:
Why not use `VERSION_GREATER_EQUAL`?

Reply:
That's not available until cmake 3.7 and the cmake minimum for this project is 3.1

Reply:
I didn't realize that was when it was introduced. Thanks for the clarification.